### PR TITLE
[DOCFIX] Add todos for improving performance tuning doc

### PR DESCRIPTION
**Pull request by bf8086**

### CODE REVIEW

**Comment by bf8086** [comment]
@calvinjia @aaudiber PTAL.

**Comment by aaudiber** [comment]
@gpang

File: docs/en/advanced/Performance-Tuning.md [hunk 1]
```diff
@@ -5,5 +5,6 @@ nickname: Performance Tuning
 group: Advanced
 priority: 0
 ---
-
-TODO(?): Move content from https://www.alluxio.org/overview/performance-tuning
+TODO(feng): Add a new section to help user identify specific bottle neck and point them to different section of the document
+TODO(feng): Move content from https://www.alluxio.org/overview/performance-tuning
+TODO(feng): Update outdated content
```
Comment (aaudiber) [inline docs/en/advanced/Performance-Tuning.md]: Do you have a sense of how which parts need to be updated/how much work there is here?

Comment (bf8086) [reply docs/en/advanced/Performance-Tuning.md]: I reviewed the doc and didn't find anything out of date. I will remove the todo item.

**Comment by bf8086** [comment]
@calvinjia I added more TODOs per your suggestion. PTAL.

**Pull request merged**